apps/client/src/app/components/transactions-table/transactions-table.component.html

```diff
@@ -177,6 +177,9 @@
         <button i18n mat-menu-item (click)="onUpdateTransaction(element)">
           Edit
         </button>
+        <button i18n mat-menu-item (click)="onDuplicateTransaction(element)">
+          Duplicate
```

Comment:
What do you think about _Clone_ as the label?

Reply:
Yep!